packages/blockchain-info-components/src/Form/SelectInput/index.js

```diff
@@ -22,10 +22,17 @@ class SelectInputContainer extends React.PureComponent {
     this.handleKeyDown = this.handleKeyDown.bind(this)
   }
 
-  componentDidUpdate (prevProps) {	
-    if (!equals(this.props.value, prevProps.value)) {	
-      this.setState({ value: prevProps.value })	
-    }	
+  componentDidUpdate (prevProps) {
+    /* eslint-disable */
+    if (!equals(this.props.value, prevProps.value)) {
+      this.setState(
+        {
+          value:
+            this.props.value
```

Comment:
@sendmebiscuits I think where we were using `nextProps`, those need to be set to use `this.props` instead of `prevProps`. I tested this one out and it was causing issues when interacting with the select component.

Reply:
You can also disable the eslint warning for setting state in cDU like the above, just make sure the logic won't get you into an infinite updating/setting state loop 😬 - ping me if you have questions!

Reply:
Nice catch phil

Reply:
Same issue in a few other places